tests/testthat/test-get_code.R

```diff
@@ -264,6 +264,23 @@ testthat::test_that("@linksto tag indicate affected object if object is assigned
   )
 })
 
+testthat::test_that("get_code works for assign detection no matter how many parametrers were provided in assign", {
+  code = c(
+    "x <- 1",
+    "assign(\"x\", 0, envir = environment())",
+    "y <- x"
+  )
+
+  tdata <- teal_data() |> eval_code(code)
```

Comment:
Should we use `|>`?

The R requirement is "only" 4.0 and it doesn't support the native pipe

I can't find the release notes, but here's a console with R 4.0.5

![image](https://github.com/insightsengineering/teal.data/assets/211358/0f2f2ff7-d243-4c6b-a577-9ad3c3c7e733)

Reply:
You are correct, I will remove pipes in #263

Reply:
will change to `eval_code(teal_data(), code)`